### DISCUSSION TITLE
Should we port taxcalc parameters and variables that aren't part of current or past law?

### DISCUSSION THREAD

**Discussion by MaxGhenis** [PolicyEngine/policyengine-us#126] [General]
OpenFisca-UK only contains parameters and values that are part of current or past law. For example, UBI is part of reforms.

I think we have three options with respect to this issue:
1. Follow OpenFisca-UK and don't port any taxcalc variables or params that aren't in current or past law. For example, all the [child and elderly care above-the-line deductions](http://taxcalc.pslmodels.org/guide/policy_params.html#child-and-elderly-care) were part of Trump's 2016 campaign platform, but (as evident by the zeros) none were ever implemented. This is basically #51.
2. Port everything from taxcalc so we can fully achieve parity, then remove these variables and params at that point, or move them to some other repo or section.
3. Port everything and allow openfisca-us to contain reform variables and params.

Let's discuss at the dev meeting tomorrow.

**Comment by nikhilwoodruff** [comment]
I think there's definitely a case for including the functionality, and it seems a good idea would definitely to incorporate reform code in some other Python module (either elsewhere in OpenFisca-US, or probably neater, in a different repo).  I'd most agree with 2), but let's discuss later.
